src/server/req_preemptjob.c

```diff
@@ -102,7 +101,7 @@ req_preemptjobs(struct batch_request *preq)
 
 		pjob->preempt_order = svr_get_preempt_order(pjob, psched);
 		pjob->preempt_order_index = 0;
-		switch((int)pjob->preempt_order[0].order[j]) {
+		switch((int)pjob->preempt_order[0].order[0]) {
```

Comment:
Won't this always use the first preemption method?  If I set preempt_order to SCR, won't it only try S and not C or R if S fails?

Reply:
trying the other methods is done in reply_preempt_jobs_request(). the variable j was not serving any purpose.